### DISCUSSION TITLE
API command to execute Tasks --> Identify (graphql)

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#5489] [Q&A]
- Command to execute (not sure about the variable Data, i.e. Graphql data):

```bash
Data='{ "query": "mutation ) }" }'
```

```bash
        Url="http://[::1]:9999/graphql"
	CommandA=(
		curl
		-X POST
		-H "Content-Type: application/json"
		--silent
		--data "${Data}"
		"${Url}"
	)
"${CommandA[@]}"
```

# Options

Identifying scenes from the following paths:    /mnt/Secure/Video/Unsort
Sources:
  - stash-box: StashDB - dirkgraf2000
  - Auto Tag

![image](https://github.com/user-attachments/assets/41e99850-1ff0-4e55-ac88-aa312ccb5953)

![image](https://github.com/user-attachments/assets/203fe4b5-9f19-4c26-aaa9-91120e073070)

**Comment by feederbox826** [comment]
I recommend you use a GraphQL explorer like Postman or Insomnia. The command is `mutation metadataIdentify`, which you will have to customize for your needs
